coilsnake/modules/eb/SkipNamingModule.py

```diff
@@ -49,7 +50,8 @@ def write_to_rom(self, rom):
             offset = self.write_loader_asm(rom, offset, self.data["Food"],  6, 0x1f, 0x98)
             offset = self.write_loader_asm(rom, offset, self.data["Thing"], 6, 0x29, 0x98)
 
+
```

Comment:
Your latest changes seem to have added an extra newline on line 53, could you please remove it?

Reply:
Fixed, and I'm drafting a comment now :)